package-lock.json

```diff
@@ -6030,6 +6088,43 @@
       "resolved": "https://registry.npmjs.org/escape-string-regexp/-/escape-string-regexp-1.0.5.tgz",
       "integrity": "sha1-G2HAViGQqN/2rjuyzwIAyhMLhtQ="
     },
+    "escodegen": {
```

Comment:
are these new packages related to burger-menu or responsiveness?

Reply:
Same reply as before